mapie/tests/test_calibration.py

```diff
@@ -106,6 +106,18 @@
     X_, y_, test_size=0.33, random_state=random_state
 )
 
+gb = GradientBoostingClassifier(
+            random_state=random_state
+            )
+
+
+def early_stopping_monitor(i, est, locals):
+    """Returns True on the 3rd iteration."""
+    if i == 2:
+        return True
+    else:
+        return False
```

Comment:
I also suggest moving it directly into the test to better understand where the function is being used.